database/models/prices.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+
+const priceSchema = new mongoose.Schema({
+  productId: { type: Number, unique: true },
+  seller: [{
+    id: Number,
+    price: Number,
+    tax: Number,
+  }],
+});
+
+module.exports.Price = mongoose.model('Price', priceSchema);
```

Comment:
An idea, maybe just assigning the model directly to module.exports instead of module.exports.price if this is the only thing that will be exported

Reply:
I will leave it as is. I intend to add DB queries, and export those as well so my controller could consume those.